pkgs/games/minetest/mods/default.nix

```diff
@@ -0,0 +1,62 @@
+{ pkgs ? import <nixpkgs> { } }:
+with pkgs;
+let buildMod = (import ./build-minetest-mod.nix { inherit stdenv; }).buildMinetestMod;
+in
+{
+
+  creaturaMod = buildMod {
```

Comment:
I think the `Mod` suffix is redundant since we are already in the minetestMods scope, and names in contentdb are unique across mods and games (not forks though)

Reply:
Fair point, I had thought about that too. I'll fix this.